apps/hyperdrive-trading/src/ui/analytics/getAnalyticsUrl.ts

```diff
@@ -0,0 +1,15 @@
+import { Address } from "viem";
+import { sepolia } from "viem/chains";
+
+export function getAnalyticsUrl({
+  chainId,
+  hyperdrive,
+}: {
+  chainId: number;
+  hyperdrive: Address;
+}): string | undefined {
+  if (chainId === sepolia.id) {
+    // Block Analytica doesn't checksum addresses in the url, so we must lowercase
```

Comment:
This comment is confusing to me. The fact that they don't validate checksummed address doesn't inherently require converting to lowercase, but doing so when you're able to is good practice.
```suggestion
    // Block Analytica doesn't validate checksummed addresses in the URL, so we can safely lowercase
```

Reply:
good call